### ISSUE TITLE
Separate libs and lua to different repos

### ISSUE THREAD

**Issue by brynne8** [luapower/openssl#1]
It's often when we only need the binary libs to run other libs. Downloading this repo would make our working directory full of openssl binding files.

**Comment by capr** [comment]
True, you get stuff you don't need, but can I justify doing it only for openssl and not for the other 50 or so libs that have both bindings and binaries in the same repo?
